src/panels/developer-tools/event/developer-tools-event.js

```diff
@@ -25,7 +25,10 @@ class HaPanelDevEvent extends EventsMixin(LocalizeMixin(PolymerElement)) {
       <style include="ha-style iron-flex iron-positioning"></style>
       <style>
         .content {
-          padding: 16px;
```

Comment:
Not everything supports things like `env`, leave this in, and if it is supported the rule below will overrule it.